src/ZendeskApi.Client/Resources/RequestResource.cs

```diff
@@ -28,7 +28,7 @@ public IResponse<Request> Get(long id)
 
         public IResponse<Request> Get(IEnumerable<TicketStatus> requestedStatuses)
         {
-            return GetAsync(requestedStatuses).Result;
+            return GetAsync(requestedStatuses).ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
@kbrimble I am not too sure if that will avoid deadlock. I am testing that case currently in RM